src/App.tsx

```diff
@@ -11,25 +11,21 @@ import useAppState from './hooks/useAppState'
 import Board from './pages/Board'
 import Profile from './pages/Profile'
 import Notifications from './pages/Notifications'
-import Project from './pages/Project'
+import Projects from './pages/Projects'
 import ProjectDetails from './pages/ProjectsDetails'
-import ModifyProject from './pages/ModifyProject'
 
 const client = apolloClient()
 
 const Routes = () =>
   useRoutes([
     { path: '/board', element: <Board /> },
     { path: '/board/:projectId', element: <Board /> },
-    { path: '/projects', element: <Project /> },
+    { path: '/projects', element: <Projects /> },
```

Comment:
You should merge the development branch in yours because the route config has changed and you still use the old config object.